.jenkins/jenkins_buildbot_python2.sh

```diff
@@ -6,6 +6,7 @@ export THEANO_FLAGS=init_gpu_device=cuda
 
 # CUDA
 export PATH=/usr/local/cuda/bin:$PATH
+export CPATH=/usr/local/cuda/include/$CPATH
```

Comment:
It should be a colon before $CPATH (`:$CPATH` instead of `/$CPATH`) !

Same in codes added below.